setup/ci/gcb.md

```diff
@@ -73,3 +77,46 @@ if the build produces a Docker image, you can add an expected artifact of type *
 `gcr.io/my-project-id/my-application` (replacing `my-project-id` and `my-application` with
 appropriate values). You can then [use the produced image](/reference/artifacts/in-pipelines/)
 in downstream stages.
+
+## Configure a Google Cloud Build Stage
+
+To run a Google Cloud Build as part of a Spinnaker pipeline, create a stage of type *Google Cloud Build*.  Configure the
+stage by selecting the GCB account to use to run the build, and entering the [build configuration YAML](https://cloud.google.com/cloud-build/docs/build-config)
+in the provided text box:
+![](/setup/ci/gcb_config.png)
+You may also provide the build definition YAML as an artifact.
+
+In the *Produces Artifacts* section, you may supply any artifacts that you expect the build to create in order to
+make these artifacts available to downstream stages.  Google Cloud Build supports creating either GCS or Docker image
+[artifacts](https://cloud.google.com/cloud-build/docs/configuring-builds/store-images-artifacts), either of which
+will be converted to Spinnaker artifacts and injected into the pipeline on completion of the build.
+
+While your build is executing, the stage details will provide the current status of the build and a link to view
+the build logs in the Google Cloud Console:
+![](/setup/ci/gcb_status.png)
+
+## Configuration via Pub/Sub
+
+Prior to version 1.14, Spinnaker did not have built-in support for Google Cloud Build, but pipelines could be
```

Comment:
Now, as of 1.14, we do support GCB, but it's not equivalent to the former support described in this paragraph, right? Assuming I'm understanding this correctly. 
That is, this paragraph sounds like we're acknowledging the old way of doing something after saying the new way. And saying you can still do it the old way.
But aren't we doing two different things here:
* trigger from a GCBuild, via pub/sub
* spawn a GCBuild from a pipeline

If this understanding is correct, I'd change this paragraph to just introduce how to configure the trigger, removing references to before/after version number.

Reply:
Spawning a GCB build from a pipeline is completely new, so there's no equivalent way of doing it prior to version 1.14.

Triggering from a GCB build could be done before via the method in this section (directly configuring pubsub) but going forward the recommended way is to configure as above.  I wanted to leave this section here for users who are not yet on 1.14, or who are upgrading to 1.14 but already have the old way configured.

Maybe it would help if I made it more clear that this is an alternate way of configuring triggering, but there's no way to start a GCB build from Spinnaker prior to 1.14?